### ISSUE TITLE
Triad 2 reset problem

### ISSUE THREAD

**Issue by a1laserboy** [mschack/VCV-Rack-Plugins#28]
Hi,

Thank you for your great modules!

OSX 10.12.6
Version 050? Downloaded on 26th November

I found a small problem with Triad 2 where sending a reset trigger doesn't perform a reset back to the start of the sequence. I can provide a patch if needed.

Thanks

**Comment by mschack** [comment]
Thank you for the comment. I'll look into it.

**Comment by mschack** [comment]
I fixed ( or at least attempted to fix ) this one in the Nov 26th update.  If you have a version before that then try getting the latest, if not then there is still a problem.

**Comment by a1laserboy** [comment]
The version I have is from the 26th, but it could be an older build as I rely on people making mac builds. Is there a way I can tell from any files inside which version I have?

**Comment by mschack** [comment]
I believe I went from 4 to 8 saveable patterns on Nov 26

**Comment by a1laserboy** [comment]
Then I don't have the latest version. it's hard to follow on FB where the mac builds get posted by people that kindly build them. I'll have a search :) Thank you

**Comment by mschack** [comment]
Thanks.  I would like to do it on my own but don't have a MAC. Maybe some time soon I will buy small laptop just for fun.

**Comment by a1laserboy** [comment]
Mr Jeremy Wentworth (JW-Modules and organiser of the Community/Plugins manifest) is usually a good and fast builder, but I don't really want to ask.
